packages/common/src/configuration/service-configuration.ts

```diff
@@ -167,7 +167,7 @@ export class ServiceConfiguration {
                 },
                 minimumWaitTimeforScanResultQueryInSeconds: {
                     format: 'int',
-                    default: 120,
+                    default: 300,
```

Comment:
In DEV we may need short time. Can we set default to 15 sec and change config files instead?

Reply:
what config files, do we have different config file per profile? let's discuss it

Reply:
Set the value to 15 seconds to support development
Update config files here https://github.com/microsoft/accessibility-insights-service/tree/master/packages/resource-deployment/runtime-config